project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
It means it's not included in the classpath. In theory it's made to be used when you create a library and you don't want to bring transient dependencies. Now to be completely honest I copied and pasted from it the [macwire documentation :)](https://github.com/adamw/macwire#installation-using-with-sbt)

It makes sense as that library is only going to be used at compile time to modify the syntax tree (through macros), and is therefore useless at run time.